### PR TITLE
Ch setup project structure

### PR DESCRIPTION
**Pull request by Ruiru11**
#### What are the relevant pivotal tracker stories?
#161213241

### CODE REVIEW

File: v1/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+ECHO is on.
```
Comment (ghost) [inline v1/tests/__init__.py]: What's this for?

File: v1/manage.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+if __name__ == '__main__':
+    app.run(debug=True)
```
Comment (ghost) [inline v1/manage.py]: I suggest you rename this to `run.py`

File: v1/tests/__init__.py (hunk 1 above)
Comment (Ruiru11) [reply v1/tests/__init__.py]: I'll delete that on windows how I create a file from the command line it comes with that

File: v1/manage.py (hunk 2 above)
Comment (Ruiru11) [reply v1/manage.py]: Thanks will do that right away

**Pull request merged**